src/main/java/com/rocketden/main/model/Room.java

```diff
@@ -0,0 +1,24 @@
+package com.rocketden.main.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+public class Room {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    private Integer id;
+
+    private String roomId;
```

Comment:
I didn't want to make this PR any bigger, so I made a [card](https://trello.com/c/yrjzo9Ry/92-handle-unique-room-ids-keep-only-one-of-id-roomid-and-or-add-letters) in the Trello backlog so we can revisit all of changing to one of id/roomId, switching to GenerationType, and/or using alphanumeric rather than just numeric. Also, I originally had both `id` and `roomId` for this [security](https://stackoverflow.com/questions/396164/exposing-database-ids-security-risk) reason, although it may or may not really apply to us. Overall though, I think there's a decent amount to discuss about improving the room id design that it could warrant it's own task.

Reply:
Ah, I didn't think about any security problems -- that's a good point. Thanks for making the card, and I agree with adding all of those tasks and addressing them in another PR. Great response 🙂